src/runtime/virtcontainers/pkg/cloud-hypervisor/client/api/openapi.yaml

```diff
@@ -518,7 +518,7 @@ components:
     VmCounters:
       additionalProperties:
         additionalProperties:
-          format: uint64
+          format: int64
```

Comment:
nit: For a better context, it might be worth mentioning the PR (https://github.com/cloud-hypervisor/cloud-hypervisor/pull/1760) from clh in the commit message.